conf/bridge-server.conf

```diff
@@ -52,6 +52,9 @@ dev.external.id.add.limit = 10
 uat.external.id.add.limit = 100
 prod.external.id.add.limit = 100
 
+external.id.get.limit = 1
+prod.external.id.get.limit = 10
+
 external.id.lock.duration = 30000
```

Comment:
I've updated the naming and added comments to clarify that this is related to dynamo read capacity, and rate limiting queries in the DAO

(Not sure I interpreted your comment correctly)